### DISCUSSION TITLE
Is there any example of using - sandpack-client directly without sandpack-react

### DISCUSSION THREAD

**Discussion by pawansingh00** [codesandbox/sandpack#256] [Q&A] [answered]
Is there any example of using - sandpack-client directly without sandpack-reactt, this is especially helpful if We want to use it outside of React.

**Comment by danilowoz** [comment, marked as answer]
Hey! Sure you can! Here's some resource on how to implement it by yourself:
Example: https://codesandbox.io/s/sandpack-client-ycxrg
Documentation reference: https://sandpack.codesandbox.io/docs/advanced-usage/client
